.vscode/settings.json

```diff
@@ -4,9 +4,9 @@
   "editor.formatOnPaste": true,
   "emmet.showExpandedAbbreviation": "never",
   "editor.codeActionsOnSave": {
-    "source.fixAll.esbenp.prettier-vscode": true,
-    "source.fixAll.eslint": true,
-    "source.fixAll.stylelint": true
+    "source.fixAll.esbenp.prettier-vscode": "explicit",
+    "source.fixAll.eslint": "explicit",
+    "source.fixAll.stylelint": "explicit"
```

Comment:
Let's revert this as well if it's not related to X

Reply:
@ljharb @mcollina @chrisgervang   @brianleroux   Sorry, sir. I just learned that’s why I can’t understand what to do exactly to fix it, and that’s why I created a new pull request based on your suggested changes https://github.com/openjs-foundation/open-visualization/pull/25